app/controllers/api/v1/appointments_controller.rb

```diff
@@ -15,6 +15,7 @@ def merge_if_valid(appointment_params)
       NewRelic::Agent.increment_metric('Merge/Appointment/invalid_request')
       { errors_hash: { updated_at: 'Cancelled appointment cannot be updated', id: appointment_params[:id] } }
     else
+      set_default_appointment_type(record_params)
       appointment = Appointment.merge(record_params)
```

Comment:
this should happen only if the record is new

Reply:
Implemented change.